engine/src/w32printer.cpp

```diff
@@ -1360,6 +1362,8 @@ void MCWindowsPrinter::DoFinalize(void)
 	m_dc_changed = false;
 
 	MCValueRelease(m_name);
+	// SN-2015-03-18: [[ Win shutdown crash ]] Do not over-release m_name
+	m_name = NULL;
```

Comment:
`MCWindowsPrinter::~MCWindowsPrinter` calls `MCValueRelease(m_name)`, which was causing the crash.